<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SwitchAccWidget</class>
 <widget class="QWidget" name="SwitchAccWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>516</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>816</width>
    <height>516</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>816</width>
    <height>516</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SwitchAccWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <widget class="QWidget" name="FilletWidget" native="true">
     <property name="styleSheet">
      <string notr="true">border-radius:8px;
background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QWidget" name="TopWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>52</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>52</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>52</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(252, 252, 252);
border-bottom:1px solid rgb(222, 222, 222);
border-bottom-right-radius:0px;
border-bottom-left-radius:0px;</string>
      </property>
      <widget class="QLabel" name="labelTitle">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>15</y>
         <width>80</width>
         <height>18</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>18</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16px &quot;微软雅黑&quot;;
color: rgb(51, 51, 51);
border-bottom:0px;</string>
       </property>
       <property name="text">
        <string>账号切换</string>
       </property>
      </widget>
      <widget class="QPushButton" name="closeButton">
       <property name="geometry">
        <rect>
         <x>760</x>
         <y>10</y>
         <width>32</width>
         <height>28</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>28</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{border-image: url(:/EntCenterWidget/image/close.png);}
QPushButton:hover,pressed{border-image: url(:/EntCenterWidget/image/close_sel.png);};</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>72</y>
        <width>400</width>
        <height>36</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>36</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>36</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{border-radius:0px;
border-image: url(:/EntCenterWidget/image/lineEdit.png);padding-left:30px;font: 14px &quot;微软雅黑&quot;;color:rgb(153,153,153);}
QLineEdit:hover,pressed{border-radius:0px;
border-image: url(:/EntCenterWidget/image/lineEdit_sel.png);}
QLineEdit:focus{border-radius:0px;
border-image: url(:/EntCenterWidget/image/lineEdit_sel.png);}</string>
      </property>
      <property name="placeholderText">
       <string>请输入企业统一社会信用代码搜索</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelOtherAcc">
      <property name="geometry">
       <rect>
        <x>445</x>
        <y>82</y>
        <width>85</width>
        <height>14</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>85</width>
        <height>14</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>85</width>
        <height>14</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>其他账号登录</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSpace">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>118</y>
        <width>730</width>
        <height>1</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>730</width>
        <height>1</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>730</width>
        <height>1</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(222, 222, 222);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>129</y>
        <width>740</width>
        <height>360</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>740</width>
        <height>360</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>740</width>
        <height>360</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget{border-bottom-left-radius:0px;}
QListWidget::Item{background:rgb(255,255,255);}
QScrollBar::vertical{width:8px;background:rgba(0,0,0,0%);margin:0px,0px,0px,0px;}QScrollBar::handle:vertical{width:8px;background:rgba(0,0,0,25%); border-radius:4px;min-height:20;}QScrollBar::handle:vertical:hover{width:8px;background:rgba(0,0,0,50%); border-radius:4px;min-height:20;}QScrollBar::add-line:vertical{height:9px;width:8px;border-image:url(:/images/a/3.png);subcontrol-position:bottom;}QScrollBar::sub-line:vertical{height:9px;width:8px;border-image:url(:/images/a/1.png);subcontrol-position:top;}QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical{background:rgba(0,0,0,10%);border-radius:4px;};</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>240</y>
        <width>246</width>
        <height>105</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>246</width>
        <height>105</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>246</width>
        <height>105</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/EntCenterWidget/image/icon_zwxx.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="EntCenterWidget.qrc"/>
 </resources>
 <connections/>
</ui>
